apps/storybook-react/tailwind.config.js

```diff
@@ -8,4 +8,11 @@ module.exports = {
     './src/**/*.{js,jsx,ts,tsx}',
     './stories/**/*.{js,jsx,ts,tsx}',
   ],
+  theme: {
+    extend: {
+      fontSize: {
+        'l-display-md': 'var(--typography-l-display-md-font-size)',
+      },
+    },
+  },
```

Comment:
Remove this
```suggestion
```

Reply:
Updated https://github.com/MetaMask/metamask-design-system/pull/64/commits/8b1e66a35185f0a89e3537ed46a4230e0287e493